src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/XamlMarkupExtensionWriter.cs

```diff
@@ -293,12 +293,11 @@ public void WriteString(XamlMarkupExtensionWriter writer, string value)
         // XamlMarkupExtensionWriter returns to this state after a markup extension has been completed,
         // i.e. when the number of closing curly bracket "}" matches the number of opening curly bracket "{".
         // At this state, XamlMarkupExtensionWriter is ready to start writing a markup extension
-        class Start : WriterState
+        private class Start : WriterState
         {
-            static WriterState state = new Start();
-            Start()
-            {
-            }
+            private static WriterState state = new Start();
+
+            private Start() { }
```

Comment:
Is this really the standard to put empty constructor/methods in a single line ? From a quick look, this doesn't seem to be the case in WPF, WinForms or Runtime and I couldn't find any documentation about it.

It's probably best to revert these changes to separate them and be able to discuss it separately.

Reply:
Good catch, that wasn't intentional but it's a setting. Like I've said, I just ran it.

I'll manually revert this later.

Reply:
With that being said though:

https://github.com/dotnet/wpf/blob/5c87afff67404fa36431173683bc498bd8998022/src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/MsSpellCheckLib/Utils/RetryHelper.cs#L315

https://github.com/dotnet/wpf/blob/5c87afff67404fa36431173683bc498bd8998022/src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media3D/Transform3DGroup.cs#L53

There's quite a lot! But I'll revert it, not a scope of the PR.